styles.css

```diff
@@ -0,0 +1,414 @@
+*{
+    box-sizing:border-box;
+}
+
+a{
+    text-decoration: none;
+    display: inline;
+    color:#fff;
+    text-align: center;
+}
+
+.adv{
+    margin-left:1%;
+    cursor: pointer;
+    box-shadow:0 3px 10px rgba(0,0,0,0.2);
+}
+
+img.adv-img{
+    height: 380px;
+    width: 260px;
+}
+
+body{
+    margin: 0;
+    font-family: Roboto,Arial,sans-serif;
+    background-color: #f1f3f6;
+} 
+
+.categories{
+    display: flex;
+    padding: 0 7%;
+    overflow:auto;
+    max-width:1276px;
+    margin:0 auto;
+}
+.categories-section{
+    background-color: #fff;
+}
+.category-img-item{
+    width: 100px;
+    height: 100px;
+}
+
+.category-name{
+    text-align:center;
+    width:auto;
+    font-size: small;
+    font-weight: bold;
+}
+
+.category-name:hover{
+    color: #2874f0;
+}
+
+.category-item{
+    margin: 0 0.5%;
+    padding-top: 1%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    cursor: pointer;
+}
+
+.cart-icon{
+    margin: 0.5% ;
+}
+
+.deals, .other-deals{
```

Comment:
you could have reused `deals` class. Always name the class more generic if you think that they will be useful in the future.

Reply:
Previously had the same class name, but had to change some properties later on to make it responsive and hence had to give different name.